### PR TITLE
Dockerise

### PR DESCRIPTION
**Pull request by thibaudcolas**
In GitLab by @helenb on Nov 30, 2019, 12:04

Add docker functionality and documentation

### CODE REVIEW

**Comment by thibaudcolas** [comment]
In GitLab by @helenb on Dec 1, 2019, 09:58

I think we could merge this now - we tested it and added the docs.

File: docs/dev_process.md [hunk 1]
```diff
@@ -23,17 +23,32 @@ The repo includes a simple test application that can be run locally to develop t
...
+### Run a local build without docker
 We use [Poetry](https://poetry.eustace.io/docs/) to manage Python dependencies, so make sure you've got it installed.
 
 Then you can install the dependencies and run the test app:
 
 ```sh
-poetry install  # installs the library and its dependencies in editable mode
-poetry run ./runserver.sh  # runs the test app using the Django development server
+$ poetry install  # installs the library and its dependencies in editable mode
+$ poetry run ./runserver.sh  # runs the test app using the Django development server
```
Comment (thibaudcolas) [inline docs/dev_process.md]: I see the `runserver.sh` script has been removed, but it’s still mentioned here. Do we want to remove it here and inline its content instead?

```suggestion
$ poetry run 'django-admin runserver --settings=tests.settings --pythonpath=.'  # runs the test app using the Django development server
```

**Comment by thibaudcolas** [comment]
@helenb @bcdickinson I’d be very keen to merge this since it seems good to go / nearly there – do you have any concerns with it or should I do one last trial and then merge?

**Comment by helenb** [comment]
@thibaudcolas we both tested this on the hack day, and added docs - I think if you do one last test and don't spot any problems with it or the docs, it could be merged. thanks.

**Pull request merged**